script/tool/lib/src/version_check_command.dart

```diff
@@ -179,8 +186,13 @@ ${indentation}HTTP response: ${pubVersionFinderResponse.httpResponse.body}
     required GitVersionFinder gitVersionFinder,
   }) async {
     final File pubspecFile = package.childFile('pubspec.yaml');
-    return await gitVersionFinder.getPackageVersion(
-        p.relative(pubspecFile.absolute.path, from: (await gitDir).path));
+    final String relativePath =
+        path.relative(pubspecFile.absolute.path, from: (await gitDir).path);
+    // Use Posix-style paths for git.
+    final String gitPath = path.style == p.Style.windows
+        ? p.posix.joinAll(path.split(relativePath))
+        : relativePath;
```

Comment:
Yeah, I figured it didn't really matter either way, and the added code was trivial so I just included it. I thought it was actually a bit more readable this way since the condition makes it more obvious what the point is.